2022/Q4/artifacts/PoS-pseudocode/PoS-model.md

```diff
@@ -552,13 +554,25 @@ end_of_epoch()
 ```
 
 ```go
-//Cubic slashing function
-compute_final_rate(slashes)
+// Get total stake of active (consensus-participating) validators
+get_total_active_voting_power(epoch)
```

Comment:
perhaps this can be in another PR where we make updates as suggested in https://github.com/informalsystems/partnership-heliax/pull/43#discussion_r1203744479?